site-src/guides/backend-protocol.md

```diff
@@ -0,0 +1,30 @@
+# Backend Protocol
+
+Not all Gateway implementations support automatic protocol selection. Even in some cases protocols are disabled without an explicit opt-in (eg. websockets with Contour & NGINX). 
+
+When a Route's backend references a Kubernetes Service application developers can specify the protocol using `ServicePort` [`appProtocol`][appProtocol] field.
+
+For example the following `frontend` Kubernetes Service is indicating the port `8080` supports HTTP/2 Prior Knowledge.
```

Comment:
Maybe use a different name for the Service to avoid confusion with backend. An example like "store" would likely avoid that kind of confusion.